Themes/default/Calendar.template.php

```diff
@@ -824,12 +824,12 @@ function template_event_post()
 						<div class="event_options_left" id="event_time_input">
 							<div>
 								<span class="label">', $txt['start'], '</span>
-								<input type="text" name="start_date" id="start_date" maxlength="10" value="', $context['event']['start_date'], '" tabindex="', $context['tabindex']++, '" class="date_input start" data-type="date">
+								<input type="text" name="start_date" id="start_date" maxlength="10" value="', $context['event']['start_date_local'], '" tabindex="', $context['tabindex']++, '" class="date_input start" data-type="date">
 								<input type="text" name="start_time" id="start_time" maxlength="11" value="', $context['event']['start_time_local'], '" tabindex="', $context['tabindex']++, '" class="time_input start" data-type="time"', !empty($context['event']['allday']) ? ' disabled' : '', '>
```

Comment:
I know I was the one who suggested `$context['event']['start_date_local']` and `$context['event']['end_date_local']` here and elsewhere, but it now occurs to me that `$context['event']['start_date_orig']` and `$context['event']['end_date_orig']` are actually the better values to use. The `_local` and `_orig` versions are identical in every way, except that the `_orig` version shows the date in the *event's* time zone, whereas the `_local` version shows the date in the *user's* time zone. Since we want to display the date and time of the event in its own time zone in this part of the UI, the `_orig` version is what we want.

For the same reason, we should also replace `start_time_local` and `end_time_local` with `start_time_orig` and `end_time_orig`.

Sorry to give you extra work. 🙂

Reply:
Done